src/functionalTest/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionControllerTests.java

```diff
@@ -11,7 +11,6 @@
 import org.springframework.beans.factory.annotation.Value;
```

Comment:
I think we should add some tests for the System Admin path too in here. What do you think?

Reply:
Added delete subscription tests using system admin